### ISSUE TITLE
Bug: Content Length set incorrectly for GET requests

### ISSUE THREAD

**Issue by austince** [villadora/express-http-proxy#359]
The content length is set to `2`, even though there is no body. I've tracked this down to the `proxy.bodyContent` being set to an empty object, which is then translated to `"{}"`. This makes the requests invalid according to the HTTP spec, or so I'm told by a large company that has started rejecting these requests as malformed.

**Comment by austince** [comment]
Fixed this by setting the `parseBody` option to false, though I still believe that it should be either:   
a) enforced by default  
b) empty bodies should be translated to `""` instead of `{}`
c) GET requests should automatically override this to `""`

**Comment by monkpow** [comment]
Thanks for this report, I'll look into it.

**Comment by chalcedonyt** [comment]
Thanks for fixing this @brumar

**Comment by fs-projects** [comment]
Hello Everyone,


I am able to reproduce this issue in express-http-proxy version 1.6.3. My GET requests are blocked by my client due to this exact same issue. This is the exact reason they shared to me stating that the content-length header is having value 2 in your all GET requests.

For now I have used the workaround as shared above.

Can someone please tell me if this bug was ever fixed ? Or has it resurfaced again?

**Comment by rogeriochaves** [comment]
@fs-projects yeah, can confirm, I downgraded to 1.6.2 and it was also not fixed there, I've even tried [the PR linked above](https://github.com/frontapp/express-http-proxy/pull/1), but no success
